kobo/settings/base.py

```diff
@@ -349,6 +349,38 @@
         'having the system empty it automatically.',
         'positive_int_minus_one',
     ),
+    # Enable the use of validators with custom rules
+    'PASSWORD_VALIDATION_CUSTOM_RULES_ENABLED': (
+        True,
+        'Enable custom password validation rules. '
+        'The subsequent rules will not work unless this field is Enabled',
+    ),
+    # Built-in password validator
+    'MINIMUM_LENGTH_VALIDATION': (
+        10,
```

Comment:
How do we handle in django disabling such rule? Do we simply require superuser to type `0` to disable it?